torch_lr_finder/lr_finder.py

```diff
@@ -198,6 +205,10 @@ def range_test(
 
         print("Learning rate search finished. See the graph with {finder_name}.plot()")
 
+    def _set_learning_rate(self, new_lr):
```

Comment:
It would be nice to add some code for conversion and check, e.g.

```python
def _set_learning_rate(self, new_lrs):
    if not isinstance(new_lrs, list):
        new_lrs = [new_lrs]*len(self.optimizer.param_groups)
    if len(new_lrs) != len(self.optimizer.param_groups):
        raise ValueError(
            "Length of `new_lrs` does not equal to the number of `optimizer.param_groups`"
        )

    for i, param_group in enumerate(self.optimizer.param_groups):
        param_group["lr"] = new_lrs[i]
```

A quick example for layer-wise learning rate training scheme:
```python
# file:  pytorch-lr-finder/examples/lrfinder_mnist.ipynb
# change the declaration of optimizer to the following one
optimizer = optim.SGD(
    [
        {'params': model.fc1.parameters(), 'lr': 0.01},
        {'params': model.fc2.parameters(), 'lr': 0.001}
    ],
    lr=0.001, momentum=0.5
)

# then there will be 2 `param_group`s in the optimizer
print(optimizer)
# > SGD (
# Parameter Group 0
#     lr: 0.01
#     ... Omitted below
# Parameter Group 1
#     lr: 0.001
#     ... Omitted below
# )

# if there is only 1 `start_lrs` is given, `lr` in all `param_group` will be set as the same value
lr_finder.range_test(trainloader, end_lr=10, num_iter=100, step_mode="exp", start_lrs=1e-5)

# if multiple `start_lrs` is given, each value in `start_lrs` will be set into each `param_gruop`
lr_finder.range_test(trainloader, end_lr=10, num_iter=100, step_mode="exp", start_lrs=[1e-5, 1e-6])
```

Reply:
Good suggestion. Will make the change